### ISSUE TITLE
alignment of unison dotted quarter notes in different layers

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#746]
Unison notes in different layers that both have augmentation dots are not aligned correctly:

<img width="249" alt="screen shot 2018-02-02 at 12 35 08 am" src="https://user-images.githubusercontent.com/3487289/35723939-e814be34-07b0-11e8-930d-9d9c516f143b.png">

Test data:

```
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2018-02-02T00:35:13" version="2.0.0-dev-46b6b2e">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000001198408260">
                        <staffGrp xml:id="staffgrp-0000001685536329">
                            <staffDef xml:id="staffdef-0000000885304486" clef.shape="G" clef.line="2" n="1" lines="5">
                                <label xml:id="label-0000001072466290">    </label>
                            </staffDef>
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000000938307936">
                        <measure xml:id="measure-L3" n="1">
                            <staff xml:id="staff-L3F1N1" n="1">
                                <layer xml:id="layer-L3F1N1" n="1">
                                    <beam xml:id="beam-L5F1-L6F1">
                                        <note xml:id="note-L5F1" dots="1" dur="8" oct="4" pname="a" accid.ges="n" />
                                        <note xml:id="note-L6F1" dur="16" oct="5" pname="c" accid.ges="n" />
                                    </beam>
                                </layer>
                                <layer xml:id="layer-L5F2N2" n="2">
                                    <beam xml:id="beam-L5F2-L6F2">
                                        <note xml:id="note-L5F2" dots="1" dur="8" oct="4" pname="a" accid.ges="n" />
                                        <note xml:id="note-L6F2" dur="16" oct="4" pname="g" accid.ges="n" />
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Fixed 67216aa84aaba33d2f5e4d426e0d1d609af494d0

**Comment by craigsapp** [comment]
Looks great!:

<img width="302" alt="screen shot 2018-04-20 at 10 42 34 am" src="https://user-images.githubusercontent.com/3487289/39065695-845c3b60-4487-11e8-9745-d8c0b0faca1f.png">

<img width="499" alt="screen shot 2018-04-20 at 10 43 24 am" src="https://user-images.githubusercontent.com/3487289/39065729-a2f425b0-4487-11e8-90c6-ad1be0371c20.png">
